format/simple.js

```diff
@@ -0,0 +1,31 @@
+/* eslint no-undefined: 0 */
+'use strict';
+
+const { format } = require('logform');
+const { MESSAGE } = require('triple-beam');
+const jsonStringify = require('fast-safe-stringify');
+
+/*
+ * function simple (info, key)
+ * An adjusted version of logform's simple() format.
+ * Contains a different format that includes the timestamp.
+ * key is the metadata key, if exists.
+ *
+ * src: https://github.com/winstonjs/logform/blob/master/simple.js
+ */
+module.exports = format((info, key = '') => {
+  const stringifiedRest = jsonStringify(Object.assign({}, info, {
```

Comment:
> Also, I'm wondering, can't Elastic's officials helpers help us here? https://github.com/elastic/ecs-logging-js

because it didn't exists last month, first time im seeing this xD

Reply:
will check what it adds